packages/react-components/src/components/FileUploadCards.tsx

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Icon } from '@fluentui/react';
+import React from 'react';
+import { FileCard } from './FileCard';
+import { FileCardGroup } from './FileCardGroup';
+import { extension, truncatedFileName } from './utils';
+
+/**
+ * @beta
+ */
+export interface FileUploadCardsProps {
+  /* @conditional-compile-remove(file-sharing) */
+  /**
+   * Optional array of active file uploads where each object has attibutes
+   * of a file upload like name, progress, errormessage etc.
+   * @beta
+   */
+  activeFileUploads?: ActiveFileUpload[];
+  /**
+   * Optional callback to remove the file upload before sending by clicking on
+   * cancel icon.
+   * @beta
+   */
+  cancelFileUpload?: (fileId: string) => void;
+}
+
+/**
+ * Attributes required for SendBox to show file uploads like name, progress etc.
+ * @beta
+ */
+export interface ActiveFileUpload {
```

Comment:
This prop is being used in SendBox as well.
Let's consolidate it's definition in a single place instead of creating it twice.

Reply:
Done